cli/version/version.go

```diff
@@ -17,15 +21,75 @@ var (
 )
 
 const (
-	unknownVersion = "<unknown>"
-	cliName        = "Tarantool Cartridge CLI"
+	unknownVersion        = "<unknown>"
+	cliName               = "Tarantool Cartridge CLI"
+	cartridgeVersionTitle = "Tarantool Cartridge"
+	errorStr              = "Failed to get the version of the Cartridge"
```

Comment:
This variable name is very bad =) It doesn't say anything about this error and where it should be used.